.github/workflows/scripts/addClosingMessage.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @param {Object} params
+ * @param {import("@actions/core")} params.core
+ * @param {ReturnType<import("@actions/github").getOctokit>} params.github
+ * @param {import("@actions/github").context} params.context
+ */
+module.exports = async ({ core, context, github }) => {
+  try {
+    const owner = context.repo.owner;
+    const repo = context.repo.repo;
+    const issueNumber = context.issue.number;
+
+    const issue = await github.rest.issues.get({
+      owner,
+      repo,
+      issue_number: issueNumber,
+    });
+
+    const repositoryMap = {
+      'mui-x': 'x',
+      'material-ui': 'core',
+      'base-ui': 'base',
+      'pigment-css': 'pigment',
+      'mui-toolpad': 'toolpad',
+    }
+
+    const commentLines = [
+      ':warning: **This issue has been closed.** If you have a similar problem but not exactly the same, please open a [new issue](https://github.com/mui/mui-x/issues/new/choose).',
+      'Now, if you have additional information related to this issue or things that could help future readers, feel free to leave a comment.',
```

Comment:
You probably want to change the url here to be based on the repository this is running on?